tests/test_publish_function.py

```diff
@@ -0,0 +1,34 @@
+import math
+import os
+from fitness_functions import publish, run
+from tests import FitnessFunctionsTestBase
+from fitness_functions.publish import normalise
+
+
+class TestMetricCollection(FitnessFunctionsTestBase):
+    def setUp(self):
+        self.dummy_metrics_dict = {'dates': ['22/04/2022', '23/04/2022'], 'noqa_occurrences': [12, 24]}
+
+    def test_normalise_function(self):
+        array_for_normalisation = self.dummy_metrics_dict['noqa_occurrences']
+
+        # Formula for vector form normalization: x = sqrt(Σ𝑥2𝑖)
+        normalised_denominator = 0
+        for digit in array_for_normalisation:
+            normalised_denominator += (int(digit) ** 2)
+        normalised_denominator = math.sqrt(normalised_denominator)
+
+        # Graph plots on y axis between 0-100 so need to round and convert to whole number
+        normalised_array = []
+        for digit in array_for_normalisation:
+            normalised_array.append(round(int(digit) / normalised_denominator, 2) * 100)
+
+        normalise_data = normalise(self.dummy_metrics_dict)
+
+        self.assertEqual(normalise_data['noqa_occurrences'], normalised_array)
+
+    def test_publish_function(self):
+        mock_graph_path = os.path.join(self.mock_project_path, 'fitness/fitness_metrics_graph.png')
+        run(self.mock_project_path, self.mock_project_path)
+        publish(self.mock_project_path)
+        assert os.path.isfile(mock_graph_path) and os.access(mock_graph_path, os.R_OK)
```

Comment:
Can you please use self.assert() to be consistent with the other tests.
```suggestion
        self.assertTrue(os.path.isfile(mock_graph_path))
        self.assertTrue(os.access(mock_graph_path, os.R_OK))
```